oamapps/postConfigure/postConfigure.cpp

```diff
@@ -3971,6 +3988,14 @@ int main(int argc, char* argv[])
     cout << endl << "MariaDB ColumnStore Database Platform Starting, please wait .";
     cout.flush();
 
+    string ProfileFile;
```

Comment:
ProfileFile needs a default unless later on we treat '' as a special value meaning it's not configured later.